docs: make deprecation of classic modules more clear (#6709)
grafana/agent: docs/sources/flow/reference/components/module.string.md

title: module.string
---

# module.string (deprecated)

{{< admonition type="caution" >}}

> Modules were redesigned in v0.40 to simplify concepts. `module.string` has been deprecated as of v0.40 in favor of `import.string`, and will be removed in the following release.

Reviewer: ```suggestion
Starting with release v0.40, `module.string` is deprecated and is replaced by `import.string`.
`module.string` will be removed in a future release.
```

Looking at the wording suggested here: https://grafana.com/docs/writers-toolkit/write/deprecate-remove/ maybe a small tweak to this to make it more active voice and simplify the message in the Caution (we don't need to explain the why it was replaced).

Will it be definitely be removed in v0.41? Then we could state this explicitly.
